applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -136,3 +139,20 @@ def Initialize(self):
 
     def AdvanceInTime(self, current_time):
         raise Exception("AdvanceInTime is not implemented. Potential Flow simulations are steady state.")
+
+    def Predict(self):
+        self.solver.Predict()
+
+    def FinalizeSolutionStep(self):
+        (self.solver).FinalizeSolutionStep()
+
+    def SolveSolutionStep(self):
+        self.solver.Clear()
+        self.solver.InitializeSolutionStep()
```

Comment:
@philbucher is it ok that we do this here? When moving the wake new dofs appear that need to be added to the dof list which was the reason why we were getting segmentation fault. This means we need to create the dof list in each coupling iteration, which is not done typically.